statusboard/src/pages/Projects/Projects.tsx

```diff
@@ -20,16 +20,19 @@ import { useProjectFilters } from '../../utils/useProjectFilters';
 import { Project } from '../../utils/types';
 import getTableColumns from './utils';
 import queryParamFilter from '../../components/ProjectsTable/QueryParamFilter';
+import './Projects.scss'
 
 function Projects(): JSX.Element {
   const { allTopics, loading } = useContext(BrigadeDataContext);
   const [rowCounter, setRowCounter] = useState(0);
+  const [searchTerm,setSearchTerm] = useState("");
```

Comment:
Same as the other comment, about stashing/shelving changes

Reply:
Done